draft-ietf-moq-transport.md

```diff
@@ -952,12 +976,20 @@ When a relay receives an incoming ANNOUNCE for a given namespace, for
 each active upstream subscription that matches that namespace, it SHOULD send a
 SUBSCRIBE to the publisher that sent the ANNOUNCE.
 
-Object headers carry a short hop-by-hop `Track Alias` that maps to
-the Full Track Name (see {{message-subscribe-ok}}). Relays use the
-`Track Alias` of an incoming Object to identify its track and find
-the active subscribers for that track. Relays MUST forward Objects to
-matching subscribers in accordance to each subscription's priority, group order,
-and delivery timeout.
+If a relay receives a Publisher-Chosen Subscription ID in SUBSCRIBE_OK, it
+SHOULD assign this ID to downstream subscriptions for the same track. Since
+relays can aggregate tracks from uncoordinated publishers, it is not always
```

Comment:
```suggestion
SHOULD assign this ID to downstream subscriptions for the same track. Since
subscribers can request tracks from uncoordinated publishers through a single
relay session, it is not always
```